sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -645,7 +646,8 @@ def _set_access_control_internal(
                             name=failure.name,
                             is_directory=failure.type == 'DIRECTORY',
                             error_message=failure.error_message) for failure in resp.failed_entries],
-                        continuation=last_continuation_token))
+                        continuation=last_continuation_token  # type: ignore [arg-type]
```

Comment:
What is the type error here (assuming the same as below too)? Can it be fixed?

Reply:
```
last_continuation_token = None
current_continuation_token = None
```

Both variables are initialized to `None` so I think that's why `mypy` thinks it's `Optional[Any]`.

I looked into lines 628-631.
```python3
current_continuation_token = headers['continuation']

if current_continuation_token is not None:
    last_continuation_token = current_continuation_token
```

This is the only time `current_continuation_token` and `last_continuation_token` are assigned. From the debugger, `current_continuation_token` is either `None` or of `str` type. However, `continuation_token` keyword to `AccessControlChanges` is expected to be `str`.

I initially thought just an ignore might be the best way to solve it, but definitely open to more ideas.

Reply:
Does `AccessControlChanges` handle `None`? It seems like that type hint may be wrong? Is that something we own?